### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by justincely**
Prettied up the documentation and README.  Still needs work and information, but the auto-api is now up and the README has more information about contributing and finding more information.

Build of the docs are in /user/ely/temp/html if anyone wants to look at the local view.

### CODE REVIEW

**Comment by pllim** [comment]
Your latest commit message... I am just going to leave this here... https://xkcd.com/1296/

**Comment by justincely** [comment]
@pllim i literally had that in my head as i frustratingly banged out commit messages...turns out rebasing lets me have bad practice since i know i am just clobbering it later...

anywho, i've rebased against susan's changes and got the builds to pass.  Docs are in the same place for you to view - can you do a review and merge if ready?

File: docs/Makefile [hunk 1]
```diff
@@ -37,7 +37,6 @@ help:
 
 clean:
 	-rm -rf $(BUILDDIR)
-	-rm -rf api
```
Comment (pllim) [inline docs/Makefile]: Why is this line removed? This is a generated folder, so a clean up should delete this.

File: mosviz/loaders/test/sample_test.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import
+
+def test_sample():
+    assert(1==1)
```
Comment (pllim) [inline mosviz/loaders/test/sample_test.py]: What is the purpose of this test? If this is just a placeholder, you can also just do `assert True` or `pass`.

File: docs/index.rst [hunk 3]
```diff
@@ -7,10 +7,23 @@ The NIRSpec micro-shutter array (MSA) will produce ~100 spectra per pointing.  M
...
    readingindata
-   api
+
+API
+===
+
+.. toctree::
+   :maxdepth: 2
+
+   api/index
```
Comment (pllim) [inline docs/index.rst]: This is the wrong way to go about API doc using Astropy template. I think Susan accidentally deleted `api.rst` in #44. Just add that file back in, undo this change (i.e., `api` should be under `toctree`), and then within `api.rst`, you can do `automodapi` or whatever suits your needs for pulling in docstrings.

File: docs/Makefile (hunk 1 above)
Comment (justincely) [inline docs/Makefile]: This was just so it wouldn't clobber the api/ folder that i had put in.  I'll add this back in with the new autoapi now that it's working

File: docs/index.rst (hunk 3 above)
Comment (justincely) [inline docs/index.rst]: roger - didn't know about automodapi, and after many hours of frustration (not it's fault) seems to be working

File: mosviz/loaders/test/sample_test.py (hunk 2 above)
Comment (justincely) [inline mosviz/loaders/test/sample_test.py]: The travis build reported a failure if there wasn't an actual test running.  i.e. 0 tests != passing tests.  So i put one in there just so travis wouldn't show failures.  I'll make it a little more realistic, so we're not lying to ourselves - even if it just tests an import or something

Comment (justincely) [inline mosviz/loaders/test/sample_test.py]: NVM - i'm going to leave this as a placeholder for now.  Real tests need FITS file creation/destruction routines, and i won't be able to get to that today.

**Comment by pllim** [comment]
Looks good now. Are you still working on this? If not I can merge it.

**Comment by justincely** [comment]
nope - ready for merge.  Thanks
